src/js/app.js

```diff
@@ -0,0 +1,60 @@
+var picArray = [];
+var count = 0;
+
+$(document).ready(function(){
+  $('#inputForm').focus();
+
+  $('#searchForm').on('submit', function(e){
+    e.preventDefault();
+    $('#inputForm, #searchButton, #title, #description').addClass('hidden');
+    $('#stopButton').addClass('shown');
+
+    $.get('http://api.giphy.com/v1/gifs/search', {
+      q: $('#inputForm').val(),
+      api_key: 'dc6zaTOxFJmzC'
+    }).done(function(data){
+      var array = data.data;
+
+      for(var i = 0; i < array.length; i++) {
+        var pic = array[i].images.original.url;
+        picArray.push(pic);
+      }
+    }).fail(function(data){
+
```

Comment:
I think I left this out on Derek's code, but it's recommended to put something in these functions, whether it be a `console.log`, or more commonly, a message for the user